CONTRIBUTING.md

```diff
@@ -330,4 +330,5 @@ module.export = [
   "length": //Integer representing number of steps in the metaModule
   }
 ]
-```
\ No newline at end of file
+```
+The length is absolutely required for a meta-module, since sequencer is optimized to re-run minimum number of steps when a step is added in the UI which is 1 in the case of normal modules, if the added step is a meta-module the length of the sequence governs the number of steps to re-run.
```

Comment:
So with meta modules that don't get expanded into multiple main sequencer steps, but keep them internal only, this requirement might go away?

Reply:
@jywarren Then that meta-module would just be a normal module, so yeah no need then :D